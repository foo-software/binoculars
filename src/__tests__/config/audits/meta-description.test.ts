import auditMetaDescription from '../../../config/audits/meta-description';

describe('meta-description audit', () => {
  it('should match snapshot', () => {
    expect(auditMetaDescription('en')).toMatchSnapshot();
  });
});
