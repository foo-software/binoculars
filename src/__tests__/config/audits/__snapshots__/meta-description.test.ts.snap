// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`meta-description audit should match snapshot 1`] = `
Object {
  "description": "Meta descriptions may be included in search results to concisely summarize page content. [Learn more](https://web.dev/meta-description/).",
  "title": "Document has a meta description",
}
`;
